frame/clearing-house/src/math.rs

```diff
@@ -99,35 +99,18 @@ where
 	}
 }
 
-// The following types are markers to disambiguate the two implementations of IntoBalance below.
-// Since traits can overlap, the 'where' clauses below are not enough to tell the compiler which
-// implementation of the trait it should use for a particular type
-pub struct Int;
-pub struct FixedPoint;
-
-pub trait IntoBalance<Balance, Marker> {
+pub trait IntoBalance<Balance> {
```

Comment:
nice that you were able to remove this hack in a really clean way! :+1:

Reply:
Yeah, I was relieved to do it 😅